AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/TaskTests/UserBehaviourTests/UserBehaviorResendCodeTests.swift

```diff
@@ -13,17 +13,17 @@ import XCTest
 import AWSCognitoIdentityProvider
 import ClientRuntime
 
-class UserBehaviorResendCodeTests: BasePluginTest {
+class SendUserAttributeVerificationCodeTests: BasePluginTest {
```

Comment:
this file should also be renamed to `SendUserAttributeVerifciationCodeTests.swift`